flink-runtime/src/main/java/org/apache/flink/runtime/metrics/util/MetricUtils.java

```diff
@@ -81,7 +80,7 @@ public static ProcessMetricGroup instantiateProcessMetricGroup(
 			final Optional<Time> systemResourceProbeInterval) {
 		final ProcessMetricGroup processMetricGroup = ProcessMetricGroup.create(metricRegistry, hostname);
 
-		createAndInitializeStatusMetricGroup(processMetricGroup);
+		instantiateStatusMetricGroup(processMetricGroup);
```

Comment:
I get the struggle between git history spam vs minor code improvement. To be honest, that one is really only a cosmetic change. I realized recently that this is something I have to work on. I'm gonna revert it.

For `getActiveTaskAllocationIdsPerJob` change, you're referring to, I would rather stick to it. The method name is not matching the actual implementation. Here, the readability of the code improves considering that the return value does not help as a hint either since it's just returning `AllocationID`. I'm gonna fix the commit message prefix when reorganizing the commits.